packages/gatsby/src/redux/machines/page-component.ts

```diff
@@ -125,11 +162,14 @@ module.exports = Machine(
       }),
       deletePage: assign({
         pages: (ctx, event) => {
-          ctx.pages.delete(event.page.path)
+          if (event.page) {
+            ctx.pages.delete(event.page.path)
+          }
           return ctx.pages
         },
       }),
-      setBootstrapFinished: assign({
+      setBootstrapFinished: assign<IContext>({
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
this `// eslint-disable-next-line` doesn't seem to be needed? I don't get eslint warnings after removing it